_caseStudies/2022-10-07-oss-usage-east-africa.md

```diff
@@ -0,0 +1,92 @@
+# Open Source Software (OSS) Usage in East Africa
+### Chndy Rogel
+
+An online survey was conducted to develop understanding of data collection, analysis, visualization, and reporting tools that monitoring, evaluation, research, and learning (MERL) practitioners in East Africa use. The survey also sought to determine the rationale for choosing those tools, their effectiveness and efficiency, as well the barriers to using alternative tools. The research also aimed to determine familiarity in the usage of open source software (OSS) among respondents.
+While overall OSS adoption and usage in Africa has increased in the last two decades, the continent continues to lag behind other regions. Several early studies have placed Africa behind other geographic regions (See reference 1-3). Most recently, a [2022 study](https://www.sciencedirect.com/science/article/pii/S0040162522000105) on the geography of open source software found that no African country was in the top 30 countries of active OSS contributors on GitHub while only one African country was in the top 50 countries ranked by number of OSS developers per capita. Very few studies have been conducted on a sub-regional level. This article summarizes the result of a small study which can serve as the basis of future research on use of OSS across Africa.
```

Comment:
I would remove "the continent continues to lag behind other regions" and combine the first two sentences of the paragraph.